From 376d7b0a6209dff0e1d0136e3cc88e494bec6282 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 13 Jun 2022 10:30:23 +0300
Subject: [PATCH] [SW-0] gaudi3: pcie: remove boot termination enable cfg

Since boot termination feature is by default enabled, this patch
removes boot termination enable configuration from pcie routine
of rom recovery.

Change-Id: I312c2603fb93f13b139068dd1cba602ce5f4fb5b
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 2d0584c..358137c 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -321,7 +321,6 @@
 
 void pcie_config_boot_termination(void)
 {
-	regmap_writel(D0_PCIE_AUX_BOOT_TERM_EN_VAL_M, mmD0_PCIE_AUX_BOOT_TERM_EN);
 	regmap_writel(D0_PCIE_AUX_BOOT_TERM_CLR_CLR_M, mmD0_PCIE_AUX_BOOT_TERM_CLR);
 }
 
